### PR TITLE
Open in new tab when searching has bad contrast in dark mode 

### LINKED ISSUE #2266: Open in new tab when searching has bad contrast in dark mode
**Describe the bug**

The contrast of the open in new tab button is low when searching in dark mode

**Expected behavior**

The contrast should be high. See [this tool](https://contrast-ratio.com/) for help finding a good contrast ratio.

**Steps to reproduce the behavior:**
1. Open a book.
2. Search for something.
3. See that the open in new tab icon is still dark gray

**Screenshots**

117054862_288900635726869_1081277091660936517_n
117072271_1201215523592418_916918422522221148_n

### PR DESCRIPTION
**Pull request by 5hahryar**
Fixes #2266

Added drawable resource for night mode

**Screenshots** 
[Click here](https://ibb.co/9gwjRrP)

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.5.1-all.zip
```
Comment (macgills) [inline gradle/wrapper/gradle-wrapper.properties:6]: upgrading the gradle wrapper should be a separate ticket

File: core/src/main/res/drawable/ic_open_in_new_24dp.xml [hunk 2]
```diff
@@ -16,8 +16,10 @@
   ~
   -->
 
-<vector android:height="24dp"
+<vector xmlns:tools="http://schemas.android.com/tools"
+  android:height="24dp"
     android:viewportHeight="24" android:viewportWidth="24"
     android:width="24dp" xmlns:android="http://schemas.android.com/apk/res/android">
-    <path android:fillColor="#565656" android:pathData="M19,19H5V5h7V3H5c-1.11,0 -2,0.9 -2,2v14c0,1.1 0.89,2 2,2h14c1.1,0 2,-0.9 2,-2v-7h-2v7zM14,3v2h3.59l-9.83,9.83 1.41,1.41L19,6.41V10h2V3h-7z"/>
+    <path android:fillColor="?colorOnSurface" android:pathData="M19,19H5V5h7V3H5c-1.11,0 -2,0.9 -2,2v14c0,1.1 0.89,2 2,2h14c1.1,0 2,-0.9 2,-2v-7h-2v7zM14,3v2h3.59l-9.83,9.83 1.41,1.41L19,6.41V10h2V3h-7z"
```
Comment (macgills) [inline core/src/main/res/drawable/ic_open_in_new_24dp.xml:23]: I remember us having issues using attributes in vectors, the solution will probably be to add a resource to the night drawables

File: gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -1,6 +0,0 @@
-#Tue Feb 25 11:46:45 IST 2020
-distributionBase=GRADLE_USER_HOME
-distributionPath=wrapper/dists
-zipStoreBase=GRADLE_USER_HOME
-zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.2-all.zip
```
Comment (macgills) [inline gradle/wrapper/gradle-wrapper.properties:6]: this file is very necessary, please restore it

**Comment by 5hahryar** [comment]
I upgraded the Gradle version from 6.2 to 6.5.1 because the project couldn't build successfully.
It should be ok now.

**Comment by macgills** [comment]
I can't say for sure what you have done locally but the content of this PR is just 2 icons with no changes to the gradle wrapper and I am very happy for it to remain so. I will test it shortly

**Comment by Frans-Lukas** [comment]
Looks good to me. @macgills @5hahryar

**Comment by macgills** [comment]
Thank you for the contribution @5hahryar

**Pull request merged**